scout/server/blueprints/institutes/controllers.py

```diff
@@ -64,7 +65,19 @@ def populate_institute_form(form, institute_obj):
                 )
                 form.pheno_groups.choices.append((custom_group, custom_group))
 
-    return default_phenotypes
+    # populate gene panels multiselect
+    available_panels = list(
+        store.gene_panels().sort(
+            [("panel_name", pymongo.ASCENDING), ("version", pymongo.ASCENDING)]
+        )
+    )
+    panel_choices = []
+    for panel in available_panels:
+        panel_id = str(panel["_id"])
+        n_genes = len(panel.get("genes", []))
+        panel_name = f"{panel['panel_name']} ({str(panel['version'])}) - {str(n_genes)} genes"
+        panel_choices.append((panel_id, panel_name))
```

Comment:
I'm using panel_id (ObjectID) here because I want all specific panel objects, not the latest version of a panel.